lib/bundler/cli/add.rb

```diff
@@ -21,6 +21,7 @@ def run
 
     def perform_bundle_install
       Installer.install(Bundler.root, Bundler.definition)
+      Bundler.load.cache if Bundler.app_cache.exist? && !Bundler.frozen_bundle?
```

Comment:
Why is the `frozen_bundle?` check needed?

Reply:
🤔  I thought that it was needed because It makes the same call on https://github.com/bundler/bundler/blob/master/lib/bundler/cli/install.rb#L66.

But looking again, I think there's no need to call it from here. I'll remove the extra check.

Reply:
I see, I'm not sure it's needed there either. I'll look into it and remove the check from `bundle install` too separately.